index.html

```diff
@@ -304,29 +304,23 @@ <h3>Full name</h3>
 							<div class="custom-card">
 								<div class="avatar-wrapper">
 									<img
-										src="https://media.istockphoto.com/vectors/thumbnail-image-vector-graphic-vector-id1147544807?k=6&m=1147544807&s=612x612&w=0&h=8CXEtGfDlt7oFx7UyEZClHojvDjZR91U-mAU8UlFF4Y="
-										alt=""
+										src="https://www.dropbox.com/scl/fi/pnrvnjuut6xkb8uowzfcd/IMG_5631.jpeg?rlkey=c6184kajayoddfyp7wja5xzv2&dl=0"
+										alt="image"
```

Comment:
Aici pare ca src-ul imaginii ne duce pe site-ul de la dropbox, si nu catre sursa imaginii - de aceea img-ul nostru nu se incarca. Eu am gasit-o aici: `https://ucca09b54d25773d488388bfa8bb.previews.dropboxusercontent.com/p/thumb/ACKr48hPVWbj9gLLRa8hjHAgT_2swxZs1Q5M7gj3DWpJseG-tieF5C7Lo8jwKEFnBRXTByFdlAceYcNkOpw_V3XByt1ClcrshoiZ8wRCy1UygvGoRaGL91po-2p1cYdSfQEOppsuJW1NS4fUncuDYaNekLazU5Oh96ujVSD8hIPWnb0xtA6Nb1PsDT05YFaY7HzzbBmjtwYDYij8WK5czSYl-G8SJhnSfkpUN8TeumlyQfG_ehCl12NqwBf0fW47R0jJffxvDy5rTSIy4_WSCYpFTQ2L93GMzYJC9rhrAFVlljKzlwQbZ2tqv1fpZkKE8NI8dDpP_Eh1OJhXa2DlecoepnyHzktCnftCIl9UZpaIBS5KOZlm4W_-qXrgkollF_n5yfUWyhp9T9fhclUWQjpm/p.jpeg`

Insa fiindca URL-ul este foarte lung, te sfatuiesc sa o salvezi in folder-ul assets sub denumirea 'denisa-emina.jpeg' si apoi sa schimbi src-ul imaginii:
`<img src="assets/denisa-emina.jpeg" alt="" class="avatar"/>`

Reply:
E foarte bine acum Denisa, bravo!